.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: Test application (e2e)
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Run NPM Test (e2e)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Create SSH key
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.ETS_BASTION_HOST_KEY }}
+          name: ets-bastion-host.pem
+          known_hosts: ${{ secrets.ETS_BASTION_KNOWN_HOSTS }}
+
+      - name: Create tunnel to the dev database
+        run: ssh -i ~/.ssh/ets-bastion-host.pem -L ${{ secrets.DB_PORT }}:${{ secrets.DB_REMOTE_SERVER }}:${{ secrets.DB_PORT }} ec2-user@${{ secrets.ETS_BASTION_HOST }} -N &
```

Comment:
Should ec2-user also be a variable or does that never change?

Reply:
It's the default from AWS.